### ISSUE TITLE
Accessibility: No labels or alt-text for the interface. What do these settings mean?

### ISSUE THREAD

**Issue by spaceman7777** [JonMagon/KDiskMark#178]
- Linux-distro (kernel version): 6.17.4-4-cachyos
- Desktop Environment (KDE/GNOME etc.): KDE 6.5
- Qt Version: 6.10.0
- KDiskMark Version: 3.2.0
- FIO Version: fio-3.39

### Description:
The current interface doesn't appear to have any indication of what these extremely important inputs mean or do. As such, it isn't possible for disabled people, AI's, and people without excellent guessing skills to successfully use the application.

If labels for the UI to explain what things do is a no go, then perhaps alt-text would be an acceptable alternative? As it is, the only way to figure out what these do is to read the code, which is sub-optimal. :(

<img width="862" height="677" alt="Image" src="https://github.com/user-attachments/assets/97cf3697-5a79-4fea-9536-366237dd6134" />
